Build/GetWix.ps1

```diff
@@ -0,0 +1,18 @@
+function GetWix {
+    param($target)
+
+    Write-Output "Downloading Wix to $target"
+
+    $file = [IO.Path]::GetTempFileName()

```

Comment:
If you use `GetRandomFileName()` instead of `GetTempFileName()`, you don't have to delete the file.

```powershell
$file = Join-Path $env:temp ([IO.Path]::GetRandomFileName())
```